build.gradle

```diff
@@ -1,76 +1,9 @@
-buildscript {
-    ext.versions = [
-        kotlin: '1.8.20',
-        dokka: '1.8.10',
-        conscrypt: '2.5.2'
-    ]
-
-    repositories {
-        google()
-        mavenCentral()
-    }
-
-    dependencies {
-        classpath 'com.android.tools.build:gradle:8.0.1'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:${versions.kotlin}"
-        classpath "org.jetbrains.dokka:dokka-gradle-plugin:${versions.dokka}"
-    }
-}
-
-repositories {
-    google()
-    mavenCentral()
-}
-
-apply plugin: 'com.android.library'
-apply plugin: 'kotlin-android'
-apply plugin: 'kotlin-kapt'
-apply plugin: 'org.jetbrains.dokka'
-
-android {
-    namespace "at.bitfire.cert4android"
-
-    compileSdkVersion 33
-    buildToolsVersion '33.0.2'
-
-    defaultConfig {
-        minSdkVersion 21            // Android 5
-        targetSdkVersion 33         // Android 13
-    }
-
-    compileOptions {
-        sourceCompatibility JavaVersion.VERSION_17
-        targetCompatibility JavaVersion.VERSION_17
-    }
-
-    buildFeatures {
-        aidl = true
-        dataBinding = true
-    }
-
-    lint {
-        disable 'MissingTranslation', 'ExtraTranslation'
-    }
-
-    defaultConfig {
-        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
-    }
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+plugins {
+    id 'com.android.library' version '8.0.1' apply false
+    id 'org.jetbrains.kotlin.android' version '1.8.21' apply false
+    id 'org.jetbrains.dokka' version '1.8.10' apply false
 }
 
-dependencies {
-    implementation "org.jetbrains.kotlin:kotlin-stdlib:${versions.kotlin}"
-
-    implementation 'androidx.appcompat:appcompat:1.6.1'
-    implementation 'androidx.cardview:cardview:1.0.0'
-    implementation 'androidx.lifecycle:lifecycle-extensions:2.2.0'
-    implementation 'androidx.lifecycle:lifecycle-livedata-ktx:2.6.1'
-    implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.6.1'
-    implementation 'com.google.android.material:material:1.8.0'
-    implementation "org.conscrypt:conscrypt-android:${versions.conscrypt}"
-
-    androidTestImplementation 'androidx.test:runner:1.5.2'
-    androidTestImplementation 'androidx.test:rules:1.5.0'
-    androidTestImplementation 'com.squareup.okhttp3:mockwebserver:4.10.0'
-
-    testImplementation 'junit:junit:4.13.2'
-}
\ No newline at end of file
+group = 'at.bitfire'
+version = '1.0.0'
```

Comment:
Is this the same version as in lib/build.gradle publishing? Do we need both?

Reply:
Yes, both are necessary. There should be a way to put them in the same variable or something, but I can't manage to get it working